src/Cake.Common.Tests/Unit/Tools/MSBuild/MSBuildRunnerTests.cs

```diff
@@ -115,6 +115,23 @@ public void Should_Get_Correct_Path_To_MSBuild_Version_35(MSBuildToolVersion ver
             }
 
             [Theory]
+            [InlineData(MSBuildToolVersion.VS2017, PlatformTarget.x64, false, "/Program86/Microsoft Visual Studio/2017/Enterprise/MSBuild/15.0/Bin/amd64/MSBuild.exe")]
+            [InlineData(MSBuildToolVersion.VS2017, PlatformTarget.x86, false, "/Program86/Microsoft Visual Studio/2017/Enterprise/MSBuild/15.0/Bin/MSBuild.exe")]
```

Comment:
I think that we should add some tests for community and professional as well here.